### DISCUSSION TITLE
近期规划

### DISCUSSION THREAD

**Discussion by midoks** [midoks/mdserver-web#63] [Announcements]
有人用，才有更新的动力。

1.首先兼容。基本完成。
2.重要插件兼容。【50%】
 -  [x]  PHP
 -  [X]  Openresty
 - [X] MySQL
 - [X] Redis
 - [X] Memcachd
 
3.非重要插件，看情况更新。
4.非重要插件，要剔除。【转移到其他地方】。
5.插件开发文档【我不可能开发所有的插件需求】。

希望有能力的一起开发插件和维护代码。
- 开发文档 - https://github.com/midoks/mdserver-web/wiki/%E6%8F%92%E4%BB%B6%E5%BC%80%E5%8F%91
- 其他插件 - https://github.com/orgs/mw-plugin

### tips
回忆了一下，为什么写这个。那是我逝去青春啊。😊
我之前开发这个目的。
  - 随时可以搭建一套在线开发/测试平台。
  - 科学上网。
  - 搞搞自己的小网站。

我写这个用的BT的WEB UI，因为我写前端不行。我选flask的时候，BT用的好像web.py。

关于科学上网，网络上SS,SOCK5,L2TP, V2ray,不如我的[dagger](https://github.com/midoks/dagger) 🙂️。

**Comment by aujea** [comment]
牛批

**Reply by tanpengsccd** [reply]
牛皮。下次用这个

**Comment by HuJK** [comment]
建議弄一個 github 組織，項目和插件放裡面

未來有別的維護者可以給相對的組織權限/repo權限，不用一個pr手動merge
插件開發者可以直接給插件repo的權限

**Reply by midoks** [reply]
稳定后，会考虑的。

**Comment by weiyidu** [comment]
科学上网适合人个，机场就不行了。
